Source/s_sound.c

```diff
@@ -794,6 +813,24 @@ void S_Init(int sfxVolume, int musicVolume)
    mus_paused = 0;
 }
 
+void S_ChangeMusicDevice(void)
```

Comment:
I made an attempt to change the MIDI synth in real-time. It works on Windows, but I'm not sure about this approach.